src/sql/src/session/vars/definitions.rs

```diff
@@ -961,6 +975,15 @@ pub static PG_SOURCE_SNAPSHOT_STATEMENT_TIMEOUT: VarDefinition = VarDefinition::
     true,
 );
 
+/// Sets the `wal_sender_timeout` value to use during the replication phase of
+/// PG sources.
+pub static PG_SOURCE_WAL_SENDER_TIMEOUT: VarDefinition = VarDefinition::new(
+    "pg_source_wal_sender_timeout",
+    value!(Option<Duration>; DEFAULT_PG_SOURCE_WAL_SENDER_TIMEOUT),
+    "Sets the `wal_sender_timeout` value to use during the replication phase of PG sources (Materialize)",
```

Comment:
What's the criterion behind a config var becoming a variable here vs a dyncfg? Is it whether we want to be able to set it via SQL?

Reply:
Eventually everything should become dyncfgs. Ideally everything new is a dyncfg, but in this case I didn't want to split up the PG source configuration variables between the old way and dyncfgs.